src/travel-details-screens/utils.ts

```diff
@@ -427,3 +428,16 @@ export function getIsTooLateToBookTripPattern(
 ): boolean {
   return tripPattern?.legs?.some((leg) => getIsTooLateToBookLeg(leg, now));
 }
+
+export const getShouldShowLiveVehicle = (
```

Comment:
Should it return `false` if
- cancelled?
- `aimedArrivalTime`/`actualArrivalTime` passed > 1 hour ago?

Reply:
> Should it return `false` if
> 
> * cancelled?
> * `aimedArrivalTime`/`actualArrivalTime` passed > 1 hour ago?

Regarding cancelled:
Good suggestion and that makes sense, but after looking into it now I found it slightly harder than one would think. The `activeItem` in this file has an `isTripCancelled` property, but it can't really be used as it only reflects cancellation for the specific stop place which are specified as from-stop-place in the departure details. Even though that stop may be cancelled, the rest of the departure may go as usual. So all in all, since opening departure details for a cancelled departure is probably not happening often, I'll not do anything with this now.

Regarding logic on arrival time:
There is logic handling this now in the app, it just isn't touched here. The main logic is in this expression:
```ts
  const isJourneyFinished =
    vehiclePosition?.vehicleStatus === VehicleStatusEnumeration.Completed ||
    estimatedCallsWithMetadata.every((e) => e.actualArrivalTime);  
```

Reply:
Right. Should be fine as is for the current use case then :+1: